internal/pkg/cli/init.go

```diff
@@ -221,6 +227,9 @@ func (o *initOpts) deployEnv() error {
 		// User chose not to deploy the application, exit.
 		return nil
 	}
+	if err := o.initEnv.Ask(); err != nil {
```

Comment:
Not sure if we need this. `initEnv.Ask()` is used to get `EnvName` and `EnvProfile` in an interactive way. However, `init` by default (if not pass in value using flags) is supposed to provide an easy way for users to spin up their apps easily. That's the reason why we default `EnvName` to be "test" and "EnvProfile" to be `default` since in most cases for new projects they are set to be these values.

Also we always have value for `EnvName`, the `Ask()` command is just used to get the profile name (asking for env name will be skipped).